src/state/claimables/claimables.ts

```diff
@@ -0,0 +1,73 @@
+import { Claimable } from '@/resources/addys/claimables/types';
+import { ClaimablesArgs, getClaimables } from '@/resources/addys/claimables/query';
+import { CacheEntry, createQueryStore } from '@/state/internal/createQueryStore';
+import { userAssetsStoreManager } from '@/state/assets/userAssetsStoreManager';
+import { time } from '@/utils';
+import { convertAmountToNativeDisplayWorklet } from '@/helpers/utilities';
+import { sumWorklet } from '@/safe-math/SafeMath';
+
+export type ClaimablesStore = {
+  claimables: Claimable[];
+  totalValue: string;
+};
+
+export type ClaimablesActions = {
+  markClaimed: (uniqueId: string) => void;
+};
+
+export const useClaimablesStore = createQueryStore<ClaimablesStore, ClaimablesArgs, ClaimablesActions>(
+  {
+    fetcher: async ({ address, currency }, abortController) => {
+      return await getClaimables({ address, currency, abortController });
+    },
+    params: {
+      address: $ => $(userAssetsStoreManager).address,
+      currency: $ => $(userAssetsStoreManager).currency,
+    },
+    keepPreviousData: true,
+    enabled: $ => $(userAssetsStoreManager, state => !!state.address),
+    staleTime: time.seconds(10),
```

Comment:
This is refreshing claimables every 10s, assume this isn't intentional?

Reply:
ah whoops I lowered that for testing. Will bump it back up.